web/docs/about-vast/README.md

```diff
@@ -0,0 +1,228 @@
+  access path to your security data. VAST is an [Arrow][arrow]-native engine
+  that offers unfettered high-bandwidth access so that you can bring your own
+  workloads, with your own tools, e.g., to run iterative clustering algorithms
+  or complex feature extraction in conjunction with machine learning.
+
+Unlike a heavy-weight legacy SIEM, VAST is highly embeddable so that you can
+run it everywhere: containerized in the public cloud, on bare-metal appliances
+deep in the network, or at the edge.
+
+[siem]: https://en.wikipedia.org/wiki/Security_information_and_event_management
+[arrow]: https://arrow.apache.org
+
+### VAST vs. Data Warehouses
+
+Data warehouses,
+[OLAP](https://en.wikipedia.org/wiki/Online_analytical_processing) engines, and
+time series databases seem like an appealing choice for immutable structured
+data. They offer sufficient ingest bandwidth, perform well on group-by and
+aggregation queries, come frequently with advanced operations like joins, and
+often scale out well.
+
+However, as a cornerstone for security operations, they fall short in supporting
+the following relevant use cases where VAST has the edge:
+
+- **Data Onboarding**: it takes considerable effort to write and maintain
+  schemas for the tables of the respective data sources. Since VAST is
+  purpose-built for security data, integrations for key data sources and data
+  carriers exist out of the box.
+
+- **Rich Typing**: modeling security event data with a generic database often
+  reduces the values to strings or integers, as opposed to retaining
+  domain-specific semantics, such as IP addresses or port numbers. VAST offers a
+  rich type system that can retain such semantics, supporting both
+  *schema-on-read* (taxonomies) and *schema-on-write* (transforms).
+
+- **Fast Search**: typical query patterns are (1) automatically triggered point
+  queries for tactical threat intelligence, arriving at a high rate and often in
+  bulk, of which the majority are true negatives, (2) regular expression search
+  for finding patterns in command line invocations, URLs, or opaque string
+  messages, and (3) group-by and aggregations when hunting for threats or when
+  performing threshold-based detections. Data warehouses work well for (3) but
+  rarely for (1) and (2) as well.
+
+Bottom line: data warehouses may be well-suited for raw data processing, but
+a data backbone for security operations has a lot more domain-specific
+demands. The required heavy lifting to bridge this gap is cost and time
+prohibitive for any security operations center. This is why we built VAST.
+
+## Why not VAST?
+
+VAST is the right choice when you need to analyze large amounts of security
+telemetry for specific use cases in security operations. There exist a lot of
+database systems out there, and you may ask yourself the question: why not use
+that other system instead?
+
+To help answer this question, we offer a brief guidance when other systems might
+be a better fit.
```

Comment:
The following sections don't actually match this description. Each subsection lays out how the discussed category of databases is not a good fit for the security use-case. All of those should be captioned `VAST vs. ...` and moved to the section above.

What belongs in here is:
* You require SQL
* You require RBAC
* You need a relevance score as part of your query results
* ??

Each section would then point the reader to the db-types that might be a better fit.

Reply:
Good point. I took your approach and reworked the structure a bit. Essentially the "why not VAST" section is now gone.